### PR TITLE
Add New Featured Artist: GLORYHAMMER news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎵 🔨

### CODE REVIEW

File: news/2022-10-15-new-featured-artist-gloryhammer.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "New Featured Artist: GLORYHAMMER"
+date: 2022-10-12 10:00:00 +0000
```
Comment (Walavouchey) [inline news/2022-10-15-new-featured-artist-gloryhammer.md:4]: wrong date

File: news/2022-10-15-new-featured-artist-gloryhammer.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+---
+
+Power metal unit **GLORYHAMMER**'s intergalactic journey takes a pit stop in our osu! Featured Artist collection.
+
+![](https://assets.ppy.sh/artists/311/header.jpg)
+
+Considering they were was founded by the lead vocalist of our very own [Alestorm](https://osu.ppy.sh/beatmaps/artists/304), it was only a matter of time before **GLORYHAMMER** infiltrated our Featured Artists.
```
Comment (Walavouchey) [inline news/2022-10-15-new-featured-artist-gloryhammer.md:11]:
```suggestion
Considering they were founded by the lead vocalist of our very own [Alestorm](https://osu.ppy.sh/beatmaps/artists/304), it was only a matter of time before **GLORYHAMMER** infiltrated our Featured Artists.
```

File: news/2022-10-15-new-featured-artist-gloryhammer.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/311/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Rise Of The Chaos Wizards
+
+Try out [the map from the video above](https://osu.ppy.sh/beatmapsets/1621226) hosted by stream savant [nebuwua](https://osu.ppy.sh/users/14729352) or [this multi-million play Loved map](https://osu.ppy.sh/beatmapsets/694464) hosted by [xChorse](https://osu.ppy.sh/users/2217628)!
```
Comment (Walavouchey) [inline news/2022-10-15-new-featured-artist-gloryhammer.md:27]:
```suggestion
Try out [the map from the video above](https://osu.ppy.sh/beatmapsets/1621226) hosted by stream savant [nebuwua](https://osu.ppy.sh/users/14729352) or [this multi-million-play Loved map](https://osu.ppy.sh/beatmapsets/694464) hosted by [xChorse](https://osu.ppy.sh/users/2217628)!
```

File: news/2022-10-15-new-featured-artist-gloryhammer.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/311/Space%201992%20-%20Rise%20of%20the%20Chaos%20Wizards%2FGLORYHAMMER%20-%20Questlords%20of%20Inverness%2C%20Ride%20to%20the%20Galactic%20Fortress!.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+More glorious tunes (usually not involving hammers) can be found on [**GLORYHAMMER**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/311). If stream-centric mapping is your newfound love, **GLORYHAMMER** is guaranteed to support you.
+
+Our next announcement will be a double-release of artists we're more-than-sure you already know. The [@osugame Twitter](https://twitter.com/osugame) will have all the details in a few days, so stay tuned!
```
Comment (Walavouchey) [inline news/2022-10-15-new-featured-artist-gloryhammer.md:61]:
```suggestion
Our next announcement will be a double release of artists we're more than sure you already know. The [@osugame Twitter](https://twitter.com/osugame) will have all the details in a few days, so stay tuned!
```

**Review by peppy — APPROVED** [review]

**Pull request merged**